### PR TITLE
refactor: Integrate `consistent-type-imports` in nodes-base (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: packages/@n8n_io/eslint-config/base.js [hunk 1]
```diff
@@ -412,6 +417,20 @@ const config = (module.exports = {
...
 			},
 		},
+		// @TODO Remove these exceptions
+		{
+			files: [
+				'./packages/cli/**/*.ts',
+				'./packages/workflow/**/*.ts',
+				'./packages/core/**/*.ts',
+				'./packages/node-dev/**/*.ts',
+				'./packages/editor-ui/**/*.ts',
```
Comment (netroy) [inline packages/@n8n_io/eslint-config/base.js:427]: since this is missing `.vue` files, and cypress tests, I think it'd be better to revert the changes to this file, and add the rule only in `packages/nodes-base/.eslintrc.js`

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.213.0`
